webpack.config.js

```diff
@@ -3,19 +3,26 @@ const HTmlWebpackPlugin = require("html-webpack-plugin");
 
 module.exports = (env) => {
   return {
-    mode: env.WEBPACK_SERVE ? "development" : "production",
-
+    mode: env.production ? "production" : "development",
     entry: "./src/index.js",
     output: {
       filename: "bundle.js",
-      path: path.join(__dirname, "built"),
+      path: path.join(__dirname, "build"),
     },
     module: {
       rules: [
         {
           test: /\.jsx?$/,
           exclude: /node_modules/,
-          use: "babel-loader",
+          use: ["babel-loader"],
+        },
+        {
+          test: /\.(css|sass)$/i,
+          use: ["style-loader", "css-loader", "sass-loader"],
+        },
+        {
+          test: /\.(png|svg|jpg|jpeg|gif)$/i,
```

Comment:
For svgs, please setup svgr loader. It will make it possible to use svgs as react components by just importing svg files. You can read more here - https://react-svgr.com/docs/webpack/.

Reply:
fixed